#
# Restart the node.
#
# Some nodes may require restart before the new configuration will be applied.
#
# The request should be rejected if magic_number does not equal MAGIC_NUMBER.
#

uint64 MAGIC_NUMBER = 742196058910
uint64 magic_number

---

bool ok
